src/backend/renderer/element/texture.rs

```diff
@@ -1,84 +1,695 @@
-//! TODO: Docs
+//! Element to render a texture
+//!
+//! # Why use this implementation
+//!
+//! This module provides the following elements
+//!
+//! ## Static texture
+//!
+//! [`TextureBuffer`] represents a static texture which is expected to not change.
+//! It is possible to either use a [`pre-existing texture`](TextureBuffer::from_texture) or to create the texture
+//! from [`RGBA memory`](TextureBuffer::from_memory).
+//! The [`TextureBuffer`] can be used in the smithay pipeline by using [`TextureRenderElement::from_texture_buffer`].
```

Comment:
Same problem as above, though not as bad, because this is already describes the buffer as static and the element to be explicitly part of the pipeline.

Reply:
Changed the docs and examples to show the intended usage